### PR TITLE
Add meta-smarcimx8m-extras

### PR DESCRIPTION
**Pull request by sashko**

### CODE REVIEW

**Comment by martin-ejdestig** [comment]
> Snowboy does not provide aarch64 binaries that we can use

Neither of `aarch64-ubuntu1604` nor `android/armv8-aarch64/` can be used? At first I read it as "there are no aarch64 binaries". But it looks like there is. So you tried and it failed?

**Comment by sashko** [comment]
> > Snowboy does not provide aarch64 binaries that we can use
> 
> So you built with one of them and it failed?

Yes, both of the -- linked cannot recognize format in one case, and cannot find some symbols in another.

**Comment by martin-ejdestig** [comment]
OK, perhaps modify the comment in the bbappend a bit then since it states "doesn't provide aarch64 binaries".

File: meta-smarcimx8m-extras/recipes-alexa/avs-device-sdk/avs-device-sdk_%.bbappend [hunk 1]
```diff
@@ -0,0 +1,2 @@
+# build SDK without snowboy support as it doesn't provide aarch64 binaries for Linux
```
Comment (martin-ejdestig) [inline meta-smarcimx8m-extras/recipes-alexa/avs-device-sdk/avs-device-sdk_%.bbappend]: This (and the commit message) still reads wrong to me. There are Linux aarch64 binaries (`aarch64` in the name of the folders in snowboy) but you get errors when you try to use them.

Comment (martin-ejdestig) [reply meta-smarcimx8m-extras/recipes-alexa/avs-device-sdk/avs-device-sdk_%.bbappend]: Comment here was not changed?

Comment (sashko) [reply meta-smarcimx8m-extras/recipes-alexa/avs-device-sdk/avs-device-sdk_%.bbappend]: forgot to git add :man_facepalming:

**Pull request merged**